R/refineCenters.R

```diff
@@ -0,0 +1,143 @@
+#' @title Refine Peak Centers
+#'
+#' @description
+#' This function refines the center values of a peak by weighting the y values
+#' in the neightbourhood that belong most likely to the same peak.
+#'
+#' @param x `numeric`, i.e. m/z values.
+#' @param y `numeric`, i.e. intensity values.
+#' @param p `numeric`, peak centers.
+#' @param k `integer(1)`, number of values left and right of the peak that
+#'  should be considered in the weighted mean calculation.
+#' @param threshold `double`, ratio of the maximal peak intensity. Just values
```

Comment:
Please use `double(1)` here. Also, I think it would be more appropriate to call this the *proportion* or *percentage* of the maximal peak intensity.

Reply:
Thanks, it will be *proportion*.